### DISCUSSION TITLE
Welcome to citations Discussions!

### DISCUSSION THREAD

**Discussion by RBoelter** [RBoelter/citations#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KestrelWard** [comment]
Hi! Kestrel Ward, I'm a digital services specialist with Florida Virtual Campus, the consortium that runs the Library Services for the state of florida public institutions of higher ed. We operate the State's instance of OJS for over 40 journals around the state. We have had a request to implement this plugin for one of our 40 journals, but we don't want the plugin to be pulling data for all 40 journals, we only want them to be working with the one journal. Do you know if the credentials that must be set up with Crossref and Scopus can be different for different journals on the same OJS instance? Thank you!

**Comment by mcahyoko11** [comment]
why can't it work for ojs 3.5?
